frontend/package.json

```diff
@@ -10,28 +10,24 @@
     "coverage:watch": "npm run coverage -- --watch"
   },
   "devDependencies": {
-    "@shopgate/eslint-config": "^6.0.0",
-    "@shopgate/pwa-common": "^6.0.0",
-    "@shopgate/pwa-common-commerce": "^6.0.0",
-    "@shopgate/pwa-core": "^6.0.0",
-    "@shopgate/pwa-unit-test": "^6.0.0",
-    "babel-core": "^6.26.0",
-    "babel-plugin-react-transform": "^3.0.0",
+    "@shopgate/engage": "6.3.2",
+    "@shopgate/eslint-config": "6.3.2",
```

Comment:
Now this extension is stuck at this version till we update the dependencies again. Wouldn't it rather make sense the use `^6.5.0` for the version number?

Additionally the `@shopgate/engage` will be introduced with `6.5.0`. So we need to wait with merging this PR till this version is released.